README.md

```diff
@@ -1,35 +1,35 @@
 # Cubyz
-Cubyz is a 3D voxel sandbox game(aka minecraft clone).
+Cubyz is a 3D voxel sandbox game (AKA a Minecraft clone).
```

Comment:
I don't like how the `AKA` looks here, being the only word in all caps.
I think `aka` looks better, and honestly it's pretty widespread to write it in lowercase, so it shouldn't cause any confusion.

Reply:
Before adding the "AKA", I searched it to see what was proper, and what I found was that it can be wither AKA or a.k.a. (lowercase with periods). I could change it to just the lowercase, or the one with periods, whichever you would like.

Reply:
Actually let's change it completely: I think "(inspired by Minecraft)" sounds more fitting.
After all Cubyz is "More than just another minecraft clone"

Reply:
Change made.